packages/frontend/src/environments/environment.ts

```diff
@@ -4,8 +4,14 @@ export const environment = {
 	jwtTokenName: 'token',
 	redirectUrl: 'http://localhost:4200/redirect',
 	userPropertyNameInLocalStorage: 'currentUser',
-	feature: {
-		newComponents: true,
-		customRequest: false,
+	// BEGIN EE
+	firebase: {
+		apiKey: "AIzaSyBik7RRZ6S8QIpG4GqzwoF_SCNn3Dr9PPw",
```

Comment:
I'm not a firebase expert, but should this be public?

Reply:
Yes, It's has to be public